app/pages/project/index.cjsx

```diff
@@ -4,11 +4,11 @@ apiClient = require 'panoptes-client/lib/api-client'
 { Split } = require('seven-ten')
 counterpart = require 'counterpart'
 isAdmin = require '../../lib/is-admin'
-ProjectPage = require './project-page'
 ProjectTranslations = require('./project-translations').default
 { connect } = require 'react-redux';
 { bindActionCreators } = require 'redux';
 translationActions  = require '../../redux/ducks/translations';
+`import ProjectPage from './project-page';`
```

Comment:
Backticked import statements in coffeescript will break the test scripts.

Reply:
Thanks